### DISCUSSION TITLE
9.%E8%BE%93%E5%87%BA/hexo/15e88cd8c66974986443a627dc609838d1dd3971

### DISCUSSION THREAD

**Discussion by giscus** [AINightCoder/quartz#3] [Announcements]
# 9.%E8%BE%93%E5%87%BA/hexo/15e88cd8c66974986443a627dc609838d1dd3971

双链测试 test_note1 test_note2 test_note2 Markdown 示例 Markdown简易入门教程 使用方式：打开 Source Mode 就可以看到所有markdown格式，照着来用就行了 标题 # 标题一 ## 标题二 ### 标题三 #### 标题四 ##### 标题五 ###### 标题六 文本 粗体 引用 中划线 ...

http://localhost:8080/9.%E8%BE%93%E5%87%BA/hexo/15e88cd8c66974986443a627dc609838d1dd3971

**Comment by AINightCoder** [comment]
comment test
